tests/servers/test_basehttp.py

```diff
@@ -189,3 +190,27 @@ def test_broken_pipe_errors(self):
                         self.assertEqual(cm.records[0].getMessage(), msg)
                 finally:
                     server.server_close()
+
+
+@override_settings(ROOT_URLCONF="servers.urls")
+class LiveServerViews(LiveServerTestCase):
+    available_apps = []
+
+    def test_head_content_length_removed(self):
```

Comment:
This is already covered by `servers.test_basehttp.WSGIRequestHandlerTestCase.test_no_body_returned_for_head_requests`.

Reply:
tests updated